app/v2/models/MessageStatus.scala

```diff
@@ -26,7 +26,7 @@ import play.api.libs.json.Writes
 sealed trait MessageStatus
 
 object MessageStatus {
-  final case object Received extends MessageStatus
+  final private case object Received extends MessageStatus
```

Comment:
do we need to make this as ```private```?

Reply:
We don't strictly need to, it just wasn't being used elsewhere, i'll change it back